app/models/user.js

```diff
@@ -0,0 +1,28 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      first_name: {
```

Comment:
the coding style for JS in camelCase, not snake_case. We use snake_case only in the DB